odoo/addons/base/views/ir_cron_views.xml

```diff
@@ -14,6 +14,10 @@
                 </xpath>
                 <xpath expr="//button[@name='run']" position="replace">
                 </xpath>
+                <xpath expr="//div[@name='name_wrapper']" position="replace">
+                    <label for="ir_actions_server_id" string="Action Name"/>
+                    <h1><field name="ir_actions_server_id" domain="[('id', '=', -1)]" placeholder="e.g. Update order quantity"/></h1>
```

Comment:
Wow, I don't understand.  What is this domain for?
BTW there is an actual domain for "false": `[(0, '=', 1)]`.
Also, `placeholder` does not make sense for many2one fields.
Please explain the intention behind this.  It feels too much like a workaround.

Reply:
Well, you may *want* to reuse some existing `ir.actions.server`...